exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -120,13 +120,18 @@ func newPRWExporter(cfg *Config, set exporter.Settings) (*prwExporter, error) {
 
 	userAgentHeader := fmt.Sprintf("%s/%s", strings.ReplaceAll(strings.ToLower(set.BuildInfo.Description), " ", "-"), set.BuildInfo.Version)
 
+	concurrency := cfg.RemoteWriteQueue.NumConsumers
+	if enableMultipleWorkersFeatureGate.IsEnabled() {
+		concurrency = cfg.MaxBatchRequestParallelism
+	}
+
```

Comment:
It would be nice to always always use `cfg.MaxBatchRequestParallelism`, if it is set by the user.  That way, existing users can migrate from `RemoteWriteQueue.NumConsumers` to `cfg.MaxBatchRequestParallelism` right away.  If the feature gate is disabled, and a user has set `NumConsumers` to a non-default value, it would be nice to emit a warning instructing them to migrate.  WDYT?

Reply:
done. please check if you agree